### ISSUE TITLE
change jump key after nutron boosting

### ISSUE THREAD

**Issue by TomW1605** [DrParanoia/Aurora#8]
kept meaning to do this in the java app but never got around to it.

somthing where after supercharging your jump drive the jump key would be notsably difrent. eather a difrent colour or some sort of flashing. this would need to be a difrent flash to the charging one. i think making it eather blue or red and flashing eather realy fast or doing a slow pulsate would look good.

**Comment by DrParanoia** [comment]
Good idea. Also seems possible to implement since there is a `JetConeBoost` event in the journal. Need to research however if there are cases where this boost will wear off (obvious one being an Hyperspace jump).
